lib/createlocaltracks.js

```diff
@@ -72,7 +72,7 @@ function createLocalTracks(options) {
     audio: options.audio,
     video: options.video
   }).then(function(mediaStream) {
-    var mediaStreamTracks = mediaStream.getTracks();
+    var mediaStreamTracks = mediaStream.getAudioTracks().concat(mediaStream.getVideoTracks());
```

Comment:
@markandrus I would suggest that we would override the `getTracks` method in `lib/webrtc/mediastream.js`:
```javascript
...
MediaStream.prototype.getTracks = isSafari ? function getTracks() {
  return mediaStream.getAudioTracks().concat(mediaStream.getVideoTracks());
} : MediaStream.prototype.getTracks;
...
```